gov.pnnl.goss.gridappsd/src/gov/pnnl/goss/gridappsd/dto/CommunicationFaultData.java

```diff
@@ -0,0 +1,50 @@
+package gov.pnnl.goss.gridappsd.dto;
+
+import java.io.Serializable;
+import java.util.ArrayList;
+import java.util.List;
+
+import com.google.gson.Gson;
+import com.google.gson.JsonElement;
+
+import gov.pnnl.goss.gridappsd.dto.events.Event;
+
+public class CommunicationFaultData extends Event implements Serializable {
```

Comment:
This is very similar to another class -  CommOutage. Use one of them and delete the other.

Reply:
Move this to gov.pnnl.goss.gridappsd.dto.events folder if you decided to use this one.